conf.py

```diff
@@ -98,7 +98,7 @@
 
 # List of patterns, relative to source directory, that match files and
 # directories to ignore when looking for source files.
-exclude_patterns = ['.DS_Store', 'README', 'README.md', '.venv*']
+exclude_patterns = ['.DS_Store', 'README', 'README.md', '.venv*', '.tox', 'LICENSE.md']
```

Comment:
In `.circleci/config.yml` crei un vitualenv in `venv` mentre qui escludi `.venv`. Si dovrebbero allineare per evitare che sphinx provi a buildare altra roba.

Reply:
In questo modo:

  - circleci.yml crea un venv come da manuale circleci
  - conf.py comunque deve ignorare i venv, a prescindere che esistano o meno. 
  - chi non usa tox deve solo togliere .tox e può lanciare tutti i comandi direttamente nel circleci

L'idea di fondo quindi era di minimizzare il delta rispetto all'originale.

e se facessimo 

```
exclude_patterns = [ '.*' ]
```

Reply:
Ok potrebbe essere `exclude_patterns = [ '.*', 'venv' ]`. Altrimenti continua a buildare i file dentro il `venv` creato da circleci.